setup.py

```diff
@@ -238,8 +238,12 @@ def setup_package():
         FULLVERSION, GIT_REVISION = get_version_info()
         metadata['version'] = FULLVERSION
     else:
-        if len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel':
-            # bdist_wheel needs setuptools
+        if (len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel' or
+                sys.version_info[0] < 3 and sys.platform == "win32"):
```

Comment:
Just happened to look at that yesterday. At least we need to drop in lots of empty ``__init__.py`` files in test dirs. There will be other annoyances and issues. See https://github.com/numpy/numpy/issues/6257#issuecomment-147015057